client/service/src/builder.rs

```diff
@@ -186,6 +191,42 @@ pub type TLightClientWithBackend<TBl, TRtApi, TExecDisp, TBackend> = Client<
 	TRtApi,
 >;
 
+/// Construct and hold different layers of Keystore wrappers
+pub struct KeystoreParams {
+	keystore: Arc<Keystore>,
+	sync_keystore: Arc<SyncCryptoStore>,
+}
```

Comment:
I think something like `KeystoreContainer` might be better for this. @danforbes might have an idea too.

Reply:
Yes, good looking out, @expenses. `KeystoreContainer` seems reasonable. Other suggestions `KeystoreLayers`, `KeystoreWrapper`, `SyncedKeystoreWrapper`, `SyncedKeystore`...

Reply:
Renamed to `KeystoreContainer`